File: HelpSoftware/Help/Help.Web/Web/src/app/EF/Verify.ts
import { User } from "./EF";

export class Verify {
  ID_Verify: number;
  User: User;
  Created: Date;
  Token: string;
  IsVerified: boolean;
  VerifyTime: Date;

  deserialize(input): Verify {
    this.ID_Verify = input.ID_Verify;
    this.User = new User().deserialize(input.User);
    this.Created = input.Created;
    this.Token = input.Token;
    this.IsVerified = input.IsVerified;
    this.VerifyTime = input.VerifyTime;
    return this;
  }
}

File: HelpSoftware/Help/Help.EF/Entitys/Verify.cs
using System;
using System.ComponentModel.DataAnnotations;

namespace Help.EF
{
  public class Verify : IHelpEntity
  {
    [Key]
    public virtual int ID_Verify { get; set; }

    public virtual User User { get; set; }

    public virtual DateTime Created { get; set; }

    public virtual string Token { get; set; }

    public virtual Boolean IsVerified { get; set; }

    public virtual DateTime? VerifyTime { get; set; }

    [Timestamp]
    public virtual byte[] TimeStamp { get; set; }

    public string GetEntityName()
    {
      return "Verify";
    }

    public int GetID()
    {
      return ID_Verify;
    }

    public bool GetWriteHistoryEntry()
    {
      return true;
    }
  }
}